grails-app/assets/javascripts/transcribe/wildlifespotter.js

```diff
@@ -110,14 +133,20 @@ function wildlifespotter(wsParams, imagePrefix, recordValues, placeholders) {
       generateFormFields();
     });
 
-    $('#ct-container').on('change', 'select.numAnimals', function() {
+    //$('#ct-container').on('change', 'select.numAnimals', function() {
```

Comment:
Not sure about the style guidelines you normally follow but I'd tend to delete commented code like this.

Reply:
I usually do too. Must have missed it. Thanks.